src/main/java/strategy/handlers/PurchaseHandler.java

```diff
@@ -0,0 +1,18 @@
+package strategy.handlers;
+
+import db.Storage;
+import model.FruitTransaction;
+import strategy.OperationHandler;
+
+public class PurchaseHandler implements OperationHandler {

```

Comment:
The class PurchaseHandler should not begin with an empty line. Please remove the redundant empty line after the class declaration.